server/views/partials/layout.njk

```diff
@@ -21,4 +21,5 @@
   <script type="module" src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/mojFrontendInit.js"></script>
+  <script src="/assets/printButton.js"></script>
```

Comment:
```suggestion
  {# Javascript block for individual pages #}
  {% block pageScripts %}
  {% endblock %}
```

Then in crmDetails.njk:-

{% block pageScripts %}
<script src="/assets/printButton.js"></script>
{% endblock %}